plugins/node/opentelemetry-instrumentation-pg/src/utils.ts

```diff
@@ -153,10 +153,19 @@ export function handleConfigQuery(
     Array.isArray(queryConfig.values)
   ) {
     try {
-      span.setAttribute(
-        AttributeNames.PG_VALUES,
-        JSON.stringify(queryConfig.values)
-      );
+      const convertedValues = queryConfig.values.map(value => {
+        if (!value) {
```

Comment:
```suggestion
        if (value == null) {
```
only null/undefined should fit here, not 0 or empty strings

Reply:
good catch. thanks!